app/controllers/media_objects_controller.rb

```diff
@@ -417,8 +417,12 @@ def update_status
     Array(params[:id]).each do |id|
       media_object = MediaObject.find(id)
       if cannot? :update, media_object
-        errors += ["#{media_object.title} (#{id}) (permission denied)."]
+        errors += ["#{media_object&.title} (#{id}) (permission denied)."]
       else
+        unless media_object.title.present? && media_object.date_issued.present?
+          errors += ["Validation failed: Title field is required., Date issued field is required."]
```

Comment:
How about making this more generic since there are a number of combinations possible?  It would probably be good to reference the item id since this controller action might be handling multiple items.
```suggestion
          errors += ["#{media_object&.title} (#{id}) (missing required fields)."]
```